core/cap-0046-09.md

```diff
@@ -110,28 +113,27 @@ have exactly the same upgrade set).
 Individual config settings are validated in the way defined by the CAPs
 introducing them.
 
-#### `ConfigUpgradeSet` acceptance
+#### Disabling Soroban Transactions
 
-In order to vote for the config upgrade set, besides validating the
-`ConfigUpgradeSet` contents, validators also need to be able to accept them
-according to their own scheduled upgrades. 
+The submission of Soroban transactions can be disabled by setting
+`ConfigSettingContractExecutionLanesV0::ledgerMaxTxCount` to 0 in under the
+`ConfigSettingID` `CONFIG_SETTING_CONTRACT_EXECUTION_LANES`. Doing this will
+lock the validators out from creating config setting upgrades using WASM
+contracts, which means the `ConfigUpgradeSet` mechanism cannot be used to
+re-enable Soroban.
 
-The validator will accept a valid nominated `ConfigUpgradeSet` when it can
-accept every updated entry in it. As a general rule, the validator should
-accept a given updated `ConfigSettingEntry` when the validator has exactly the
-same upgrade scheduled. However, the acceptance rules can be adjusted by the
-CAP introducing the config entry. For example, if entries describe CPU costs of
-executing a host function, it could be viable to allow any of the nominated
-upgrade values to be within 1% of the scheduled upgrade.
+To allow increasing `ledgerMaxTxCount` from 0, we introduce the ability to set
+the value using the traditional upgrade mechanism. There's a new `LedgerUpgrade`
+type `LEDGER_UPGRADE_MAX_SOROBAN_TX_SET_SIZE`, which contains a `uint32
+newMaxSorobanTxSetSize` to update
+`ConfigSettingContractExecutionLanesV0::ledgerMaxTxCount`.
 
-If there are ever multiple valid `ConfigUpgradeSet`s that validator can accept,
-the one that has the most entries should be picked. Ties are resolved in favor
-of the lowest hash value.
+#### Minimum values
```

Comment:
I think we should either document the minimum values here or in some other CAPs. Probably here we should just say that CAPs that introduce settings should also define the minimum acceptable values.